src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/job.py

```diff
@@ -21,14 +21,23 @@ def create_job(client, resource_group_name, account_name, transform_name, job_na
         else:
             job_input = JobInputHttp(files=files, base_uri=base_uri, label=label)
 
-    job_outputs = list(map(lambda x: JobOutputAsset(asset_name=x), output_asset_names))
+    job_outputs = list(map(lambda x: _get_asset(x), output_assets))
 
     job = Job(input=job_input, outputs=job_outputs, correlation_data=correlation_data,
               description=description, priority=priority)
 
     return client.create(resource_group_name, account_name, transform_name, job_name, job)
 
 
+def _get_asset(asset_string):
+    from azure.mgmt.media.models import JobOutputAsset
+
+    name_and_label = asset_string.split('=')
+    name = name_and_label[0]
+    label = name_and_label[1]
```

Comment:
can you use some kind of tag type like we have for correlation data for example?